### PR TITLE
feat(chat): add chat populate buttons (Issue #2837)

### PR DESCRIPTION
**Pull request by Gimir**
**Description:**

- add interactive populate buttons in chat
- add confirmation dialog for buttons
- add disable input schema option

Issues:

- Issue #2837 

**UI changes**

Снимок экрана 2024-12-24 в 20 07 10

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2846.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Gimir** [comment]
/deploy-review
>- **Deployment failed**: [running](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/549831)
>Environment URL: https://chat-ai-dial-chat-pr-2846.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2846.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2846.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2846.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2846.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/store/chat/chat.selectors.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  [rootSelector],
+  (state) => state.inputContent,
+);
+
+export const selectChatFormOptions = createSelector(
+  [rootSelector],
+  (state) => state.formOptions,
+);
+
+export const selectIsChatInputDisabled = createSelector(
```
Comment (Alexander-Kezik) [inline apps/chat/src/store/chat/chat.selectors.ts]: I think it's better to pass selected chat as a param. Importing another selectors may unexpectedly lead to a circular dependency.

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2846.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request closed without merging**